packages/react/accordion/src/Accordion.tsx

```diff
@@ -35,21 +35,27 @@ const [AccordionContext, useAccordionContext] = createContext<AccordionContextVa
 
 const ITEM_NAME = 'AccordionItem';
 
-type AccordionItemOwnProps = {
-  /**
-   * Whether or not an accordion item is disabled from user interaction.
-   *
-   * @defaultValue false
-   */
-  disabled?: boolean;
-  /**
-   * A string value for the accordion item. All items within an accordion should use a unique value.
-   */
-  value: string;
-  open: never;
-  defaultOpen: never;
-  onOpenChange: never;
-};
+type AccordionItemOwnProps = Merge<
+  Polymorphic.OwnProps<typeof Collapsible>,
+  {
+    /**
+     * Whether or not an accordion item is disabled from user interaction.
+     *
+     * @defaultValue false
+     */
+    disabled?: boolean;
+    /**
+     * A string value for the accordion item. All items within an accordion should use a unique value.
+     */
+    value: string;
+  }
+>;
+
+type AccordionItemPrimitive = Polymorphic.ForwardRefComponent<
+  Polymorphic.IntrinsicElement<typeof Collapsible>,
+  Omit<AccordionItemOwnProps, 'open' | 'defaultOpen' | 'onOpenChange'>
```

Comment:
@jjenzz I wonder if this should be up too,  as really again in my mind `AccordionItemOwnProps` contains all the props `AccordionItem` accepts apart from the DOM props.